components/Avatar/avatar.interface.ts

```diff
@@ -0,0 +1,4 @@
+export type IAvatar = {
+  userImg: string | null | undefined;
+  email: string | null | undefined;
+};
```

Comment:
filename should be Avatar.interface.ts

Reply:
also please make sure you add an index.ts file in the component and exporting interface and component as well. please check other components for reference

Reply:
Please change this file's first letter "Capital"